### PR TITLE
Chapter 12

### PR DESCRIPTION
**Pull request by tienhung79**
2. Rubocop
Screenshot from 2023-09-05 15-16-39

### CODE REVIEW

**Comment by tienhung79** [comment]
ready

File: app/views/password_resets/create.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>PasswordResets#create</h1>
+<p>Find me in app/views/password_resets/create.html.erb</p>
```
Comment (vanvtt-0952) [inline app/views/password_resets/create.html.erb]: .

File: app/views/password_resets/update.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>PasswordResets#update</h1>
```
Comment (vanvtt-0952) [inline app/views/password_resets/update.html.erb]: .

File: app/views/user_mailer/password_reset.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Password reset</h1>
+
+<p>To reset your password click the link below:</p>
+  <%= link_to "Reset password", edit_password_reset_url(id: @user.reset_token, email: @user.email) %>
+<p>This link will expire in two hours.</p>
+
+<p>
+  If you did not request your password to be reset, please ignore this email and your password will stayas it is.
```
Comment (vanvtt-0952) [inline app/views/user_mailer/password_reset.html.erb]: .

File: app/views/user_mailer/password_reset.text.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+To reset your password click the link below:
+
+<%= edit_password_reset_url id: @user.reset_token, email: @user.email %>
+This link will expire in two hours.
+If you did not request your password to be reset, please ignore this email and
```
Comment (vanvtt-0952) [inline app/views/user_mailer/password_reset.text.erb:5]: .

**Comment by tienhung79** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**